Main/FileView.cpp

```diff
@@ -26,6 +26,7 @@ FileView::FileView(BRect frame) : BView( frame, "fileView", B_FOLLOW_ALL, B_WILL
 };
 
 void FileView::AttachedToWindow() {
+/*
```

Comment:
Why was this removed?
It would be preferred to use #if 0 to disable code, and add a comment explaining why it is disabled.
Or if the code is now completely unneeded, remove it completely.